src/Raven.Studio/wwwroot/App/views/database/settings/integrations.html

```diff
@@ -39,6 +39,14 @@ <h4 class="text-elipsis margin-left">
                         <span>.</span>
                     </small>
                 </div>
+                <div class="margin-top padding padding-xs text-warning bg-warning" data-bind="visible: needsLicenseUpgrade">
+                    <small>
+                        <i class="icon-warning"></i>
+                        <span>
+                            You cannot use this feature because your license doesn't allow neither PostgreSQL integration feature nor Power BI
```

Comment:
double negative.. so:

=>
* In order to use this feature, your license must include either of the following features: PostgreSQL integration or Power BI.

or 

* This feature is not available as your license allows neither PostgreSQL integration nor Power BI features

or

* This feature is not available as your license doesn't include either the PostgreSQL integration feature or the Power BI feature

Reply:
let's go with: In order to use this feature, your license must include either of the following features: PostgreSQL integration or Power BI.

Reply:
I would go with 1'st bullet from above
We can make it shorter as follows:

`To use this feature, your license must include either of the following features: PostgreSQL integration or Power BI.`

Reply:
https://github.com/ravendb/ravendb/pull/14163